specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/confidentialledger.json

```diff
@@ -349,6 +349,36 @@
         }
       }
     },
+    "/app/subLedgerIds": {
+    "get": {
+        "operationId": "ConfidentialLedger_GetSubLedgerIds",
+        "x-ms-examples": {
+            "GetSubLedgerIds": {
+                "$ref": "./examples/ledger/GetSubLedgerIds.json"
+            }
+        },
+        "summary": "Retrieves the ids of subledgers",
+        "description": "Subledgers are what we call the keys in our key-value storage.",
```

Comment:
We don't typically talk about the underlying key-value store in customer-facing documentation. We can either omit the description or provide a higher-level description like "Sub-ledgers are user-created collections of ledger entries".

Reply:
That makes sense. Thanks for explaining.